countdown.js

```diff
@@ -1,5 +1,20 @@
-function countdown(seconds){
-  // ...
-}
+//Exercise 2 - Countdown 
+function countdown(seconds) {
+  var timer = parseInt(seconds);
+  var delay = 1000;
+  try {
+    var startTimer = setInterval(function() {
+      console.log(timer);
+      timer--;
+      if (timer == 0) {
+        clearInterval(startTimer);
+      }
+    }, delay);
 
+  } catch (e) {}
+}
 countdown(5);
+
+function cancelSetInerval() {
+  clearInterval();
+}
```

Comment:
Careful to not leave "dead code" around! Surefire way to infuriate your coworkers :smirk: